server/sttp-stub-server/src/test/scala/sttp/tapir/server/stub/TapirStubInterpreterTest.scala

```diff
@@ -204,6 +205,29 @@ class TapirStubInterpreterTest extends AnyFlatSpec with Matchers {
     response.body shouldBe Left("Internal server error")
     response.code shouldBe StatusCode.InternalServerError
   }
+
+  it should "handle multipart body" in {
```

Comment:
could you add a test which uses the multipart, to verify that it is correctly received by the endpoint's logic? The test could use `.whenServerEndpointRunLogic` in the stub.